src/components/ShortcutBox.vue

```diff
@@ -1,440 +1,412 @@
 <template>
   <div>
     <v-autocomplete
-        v-model="select"
-        :items="suggestions"
-        :search-input.sync="searchString"
-        :filter="(item, queryText, itemText) => true"
-        :menu-props="{maxHeight: 600,}"
-        item-text="displayValue"
-        return-object
-        rounded
-        :dense="dense"
-        filled
-        clearable
-        hide-no-data
-        hide-details
-        class="shortcut-box"
-        :placeholder="placeholder"
-        prepend-inner-icon="mdi-magnify"
-        ref="shortcutBox"
-        :autofocus="autofocus"
-        :loading="isLoading"
-
-        @change="onChange"
-        @click:clear="clickClear"
-        @keydown.enter="isEnterPressed = true"
-        @keyup.enter="onEnterKeyup"
-    >
-      <!--        @blur="clear"-->
-      <template v-slot:prepend-inner>
+      ref="shortcutBox"
+      prepend-inner-icon="mdi-magnify"
+      return-object
+      rounded
+      v-model="select"
+      v-model:search="searchString"
+      :items="suggestions"
+      item-title="displayName"
+      :custom-filter="(item, queryText, itemText) => true"
+      :menu-props="{ maxHeight: 600 }"
+      :density="dense ? 'compact' : 'default'"
+      :placeholder="placeholder"
+      :autofocus="autofocus"
+      :menu="menuOpen" 
+      :loading="isLoading"
+      @click:clear="clickClear"
+      @update:modelValue="onChange"
+      @keyup.enter="onEnterKeyup"
+    > 
+      <!-- Chip Slot -->
+      <template v-slot:chip> 
```

Comment:
There is a lot of very Important logic and functionality wrapped up in this element. I am lacking complete knowledge of everything that is expected of this element.  If I was undertaking this conversions, I would spend some time writing test coverage for this functionality . I would want to be confident that it continues to work it does now.